frontend/www/js/controllers/pictureController.js

```diff
@@ -14,8 +14,18 @@ playfestApp.controller('pictureCtrl', function($scope, $cordovaCamera) {
               saveToPhotoAlbum: false
           };
 
-    $cordovaCamera.getPicture(options).then(function(imageData) {
+    return $cordovaCamera.getPicture(options).then(function(imageData) {
       $scope.imgURI = "data:image/jpeg;base64," + imageData;
+      return $upload.upload({
+               url: 'http://localhost:5000/poster',
+               method: 'POST',
+               file: imageData,
+           }).progress(function(evt) {
+               console.log('percent: ' + parseInt(100.0 * evt.loaded / evt.total));
+           }).success(function(response, status, headers, config) {
+               console.log("Image uploaded!");
+               return response.data;
+           });
     }, function(err) {
       console.log("It isn't working, mate.")
```

Comment:
nice to have console log for now because we havnet checked if it works on phone with the ocr so still need to debug!